Source/VFECore/Abilities/ModExtensions/AbilityExtension_RandomPowerMultiplier.cs

```diff
@@ -4,9 +4,15 @@ namespace VFECore.Abilities;
 
 /// power (after stat offsets) becomes a random number within a range.
 ///
-/// The formula used is Rand.Range(power * min, power * max)
+/// The formula used is power * Rand.Range(range.min, range.max)
+/// Example XML:
```

Comment:
Note that there is special handling for `FloatRange`, so you can do something like `<range>0.5~1.5</range>`.

Reply:
Thanks for this! Was not aware!